bundles/org.openhab.automation.jrubyscripting/src/main/java/org/openhab/automation/jrubyscripting/internal/JRubyScriptEngineConfiguration.java

```diff
@@ -379,7 +379,7 @@ private enum Type {
         private final String defaultValue;
         private final Optional<String> mappedTo;
         private final Type type;
-        private Optional<String> value;
+        private @Nullable String value = null;
```

Comment:
It's the default value and assigning it again wastes CPU cycles.

```suggestion
        private @Nullable String value;
```

Reply:
Fixed